### DISCUSSION TITLE
未来美国EB2排期绝无可能只有3年，2022年新申请人可能面临10年排期

### DISCUSSION THREAD

**Discussion by GentlemanXu** [The-Run-Philosophy-Organization/run#139] [General]
#61 
[王探长谈为什么来了就是美国人.md](https://github.com/The-Run-Philosophy-Organization/run/blob/main/%E6%B6%A6%E5%AD%A6%E6%96%B9%E6%B3%95%E8%AE%BA/%E5%90%84%E5%9B%BD%E9%80%89%E6%8B%A9/%E7%BE%8E%E5%9B%BD/%E7%8E%8B%E6%8E%A2%E9%95%BF%E8%B0%88%E4%B8%BA%E4%BB%80%E4%B9%88%E6%9D%A5%E4%BA%86%E5%B0%B1%E6%98%AF%E7%BE%8E%E5%9B%BD%E4%BA%BA.md) 中轻描淡写地提到了 `你申请了EB2的国家利益豁免签证之后，可能需要等3年`。而实际上2022以后，中国大陆出生的EB2新申请人将面临7年至15年不等的排期

最近两年大陆出生EB2申请人的三年排期是因为
1. 吃到了亲属移民(FB)的没用掉名额(Spill Over)，导致总名额大致翻倍，中国申请人得到2x7%配额。2022年10月开始中国大陆申请人名额将恢复正常。
2. 许多人在2021左右EB3排期领先的时候从EB2降级到EB3，导致EB2需求减少，排期大进。从2019年10月至2021年10月EB3的排期是领先于EB2的，于是许多EB2申请人在此期间从EB2降级至EB3，导致EB3需求增加，EB2需求减少，EB3排期倒退，EB2排期前进。 如果先前降级至EB3的申请人重新升级回EB2将导致EB2排期停滞甚至倒退。EB2/3的升降级机制是排期大进大退的一个重要原因。

根据国会下属机构Congress Research Service 在2020年发布的[报告](https://crsreports.congress.gov/product/pdf/R/R46291)，如果没有移民改革取消 **出生地配额限制** 或者 **增加EB总名额**, 在2020年申请EB2的中国大陆出生居民将排期**18年**才能拿到绿卡。

想润美国的同学们需要认真考虑排期的问题。如果对美国身份非常看重的话可以考虑
1. 与美国公民**结婚**，Immediate Relative (IR) 无限额无排期
2. 与绿卡居民**结婚**，FB-2A中国大陆无排期
3. EB2-NIW占Priority Date, 后面**提高资历升EB1A**
4. 与**出生地EB对应类别无排期**的居民结婚，通过Cross-Chargeability来使用他/她的出生国排期以绕过EB排期
5. 法拉盛一条龙政庇

**Comment by the-run-philosophy** [comment]
他说的是EB2-NIW吧。正如你说的EB2-NIW占Priority Date, 后面提高资历升EB1A

**Comment by GentlemanXu** [comment]
> 他说的是EB2-NIW吧。正如你说的EB2-NIW占Priority Date, 后面提高资历升EB1A

是的很多人是这样做的，我其实主要想提醒的是那些未来没有太大机会升EB1A的同学
